src/core/loss_detection.c

```diff
@@ -9,10 +9,16 @@
     have been lost or delivered to the peer.
 
 
-    RACK (time-based loss detection) algorithm:
+    A combination of two similar loss detection algorithms is used:
 
-    An unacknowledged packet sent before an acknowledged packet and
-    sent more than QUIC_TIME_REORDER_THRESHOLD ago is assumed lost.
+    RACK (time-based reordering threshold) algorithm:
+        An unacknowledged packet sent before an acknowledged packet and
+        sent more than QUIC_TIME_REORDER_THRESHOLD ago is assumed lost.
+
+    FACK (packet-number-based reordering threshold) algorithm:
```

Comment:
The existing comment mentioned only RACK and made me wonder if we'd one RACK-only LD in here, which we didn't, so I added this mention of FACK.